### PR TITLE
Feature/developer booklet changes

### PR DESCRIPTION
**Pull request by Jonathan-Rosenberg**
1. Two typo fixes
2. Added a possible case if someone defined postgres with a password
3.  Running from the IDE: added the `go run` command before the file

### CODE REVIEW

File: booklet.md [hunk 1]
```diff
@@ -9,33 +9,34 @@ Follow this section to get lakeFS up and running for the first time, add some da
...
    ```
- 1. You should be able to browse to the UI from [http://localhost:8000](http://localhost:8000).
+8. You should be able to browse to the UI from [http://localhost:8000](http://localhost:8000).
 
 
 ### Goal: Run lakeFS from your IDE
 
-Open the project in your IDE (ask your buddy for an IntelliJ license), and run `cmd/lakefs/main.go`.
-This file compiles to the binary weran in the previous step. Don't forget to add the `run` argument.
+Open the project in your IDE (ask your buddy for an IntelliJ license), and run `go run cmd/lakefs/main.go`.
```
Comment (nopcoder) [inline booklet.md:38]: Think the idea here was to select the run option while selecting the specific run.go. The IDE will create a configuration profile to run and debug it.

**Review by nopcoder — COMMENTED** [review]
comment about the run itself.
markdown numbering is done automatically.
just keep them all 1.

File: booklet.md (hunk 1 above)
Comment (johnnyaug) [reply booklet.md:38]: Yes, that was the idea. I agree that it wasn't the best phrasing.
Perhaps ```...and create a run configuration for the file `cmd/lakefs/main.go`. ```

**Review by johnnyaug — CHANGES_REQUESTED** [review]
Thanks! See comment

File: booklet.md (hunk 1 above)
Comment (Jonathan-Rosenberg) [reply booklet.md:38]: 👍

**Review by nopcoder — APPROVED** [review]

**Review by johnnyaug — APPROVED** [review]

**Pull request merged**